src/signals/incident/components/IncidentPreview/components/MapSelect/MapSelect.test.js

```diff
@@ -26,15 +26,15 @@ describe('signals/incident/components/IncidentPreview/components/MapSelect/MapSe
 
   it('returns a location', () => {
     expect(getLatlng(incidentFixture.location)).toEqual({
-      latitude: incidentFixture.location.geometrie.coordinates[1],
-      longitude: incidentFixture.location.geometrie.coordinates[0],
+      lat: incidentFixture.location.geometrie.coordinates[0],
+      lng: incidentFixture.location.geometrie.coordinates[1],
     })
   })
 
   it('returns default coords', () => {
     expect(getLatlng()).toEqual({
-      latitude: DEFAULT_COORDS[1],
-      longitude: DEFAULT_COORDS[0],
+      lat: DEFAULT_COORDS[0],
+      lng: DEFAULT_COORDS[1],
     })
   })
```

Comment:
is it correct that the indexes should be reversed here?

Reply:
Yes, the default coordinates that were used in the `MapSelect` component were set in an order that differs from the usual order. Like the API, the order in the `MapSelect` component was set to `lng,lat` instead of `lat,lng`. I removed the default coordinates from the `MapSelect` implementation and replaced them with the coordinates from the configuration. The configuration uses the 'correct' order. That is why the change in the test was required.